lte/gateway/deploy/roles/magma/files/systemd/magma_ovpn.service

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[Unit]
+Description=Magma %i service
```

Comment:
Can you use the placeholder in a service file that's fully-qualified? I think you should type it all out your self anyways